game/sony/PS/PSP/psp_save_metadata.ksy

```diff
@@ -0,0 +1,102 @@
+meta:
+  id: psp_save_metadata
+  endian: le
+  encoding: utf-8
+  license: Unlicense
+  file-extension: SFO
```

Comment:
This file lacks `title` — I'd suggest to move current `doc` contents here and extend them to be more useful for an average user doing a search query, i.e. something like

```
title: Sony PlayStation Portable (PSP) save game metadata file (PARAM.SFO)
```

Ideally, I'd love to have `doc` entry too, outlining what it is all about:

* what "Sony PlayStation Portable" is,
* what is "save metadata",
* when and by whom it is created (is it a particular game? is it OS-wide thing? is it hardware-wide thing?),
* where physically it is created (is it some sort of removable memory card? some sort of flash memory built-in in the console?)
* how does one access this file (just read it from a memory card? dumps RAM of the console? dumps built-in flash? unsolders some chip and dumps it?)